### PR TITLE
Adding channels.info method

### PR DESCRIPTION
**Pull request by stedn**
Slack supports queries on the channel info.  Would be useful for discovering channel topic.

### CODE REVIEW

File: tests/unit/channels/test_info.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import slack.http_client
+
+
+slack.api_token = 'my_token'
+
+class TestChannelsInfo(unittest.TestCase):
+    @patch.object(slack.http_client, 'get')
+    def test_list(self, http_get_mock):
+        slack.channels.info('C1234567890')
+        http_get_mock.assert_called_with('channels.history', {
```
Comment (kn) [inline tests/unit/channels/test_info.py]: I think you need `channels.info` instead of `channels.history`.

**Review by kn — APPROVED** [review]
nice!

**Comment by stedn** [comment]
test is failing with `Unable to download 2.6 archive. The archive may not exist. Please consider a different version.`.

**Comment by kn** [comment]
ah feel free to drop `2.6` from `slack/.travis.yml`. I think it's time :)

**Comment by stedn** [comment]
tests pass, will it be possible merge and ship to pip?

**Comment by kn** [comment]
`0.4.0` is up now!

**Pull request merged**